PagingWithNetworkSample/versions.gradle

```diff
@@ -189,5 +189,6 @@ def addRepos(RepositoryHandler handler) {
     handler.google()
     handler.jcenter()
     handler.maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
+    handler.maven { url '/usr/local/google/home/dustinlam/google/androidx-paging-release/out/host/gradle/frameworks/support/build/support_repo' }
```

Comment:
is it just me or does this look incorrect to have in a public and open source repo?

Reply:
You are correct :blush: